src/core/js/views/buttonsView.js

```diff
@@ -96,28 +92,29 @@ define(function() {
                 var buttonText = this.model.get('_buttons')["_" + changedAttribute].buttonText;
 
                 switch (changedAttribute) {
-                    case "showCorrectAnswer": case "hideCorrectAnswer":
+                    case "submit":
+                    case "showCorrectAnswer":
+                    case "hideCorrectAnswer":
     				    //make model answer button inaccessible but enabled for visual users
     				    //	due to inability to represent selected incorrect/correct answers to a screen reader, may need revisiting
-                        this.$('.buttons-action').a11y_cntrl(false).html(buttonText).attr('aria-label', ariaLabel);
+                        this.$('.buttons-action').a11y_cntrl_enabled(false).html(buttonText).attr('aria-label', ariaLabel);
```

Comment:
just reverting your changes to these lines should do it: https://github.com/barry-learningpool/adapt_framework/blob/135dddd29245bf9456bb3f9581c7c460b43b508e/src/core/js/views/buttonsView.js#L94-L106